yarn-project/txe/package.json

```diff
@@ -18,7 +18,7 @@
     "formatting": "run -T prettier --check ./src && run -T eslint ./src",
     "formatting:fix": "run -T eslint --fix ./src && run -T prettier -w ./src",
     "test": "NODE_NO_WARNINGS=1 node --experimental-vm-modules ../node_modules/.bin/jest --passWithNoTests",
```

Comment:
There was a bug in TXE which resulted in a test passing even when an error was thrown. Grego fixed it in this PR and what follows are changes made by him.